ftp.go

```diff
@@ -47,8 +47,9 @@ type Entry struct {
 
 // Response represents a data-connection
 type Response struct {
-	conn net.Conn
-	c    *ServerConn
+	conn       net.Conn
+	c          *ServerConn
+	connClosed bool
```

Comment:
I prefer to name the field just `closed`. The name of the corresponding method is also just `Close`.